requirements.txt

```
@@ -10,12 +10,12 @@ sympy==1.4
 pillow==6.2.0
 pip==19.2
 protobuf==3.8.0
-pennylane==0.8.0
+git+https://github.com/XanaduAI/pennylane.git@#egg=pennylane
```

The PennyLane version error seems to arise from the fact that this is using an `egg` whereas the Cirq and Qiskit plugin links will build a wheel. Due to this fact, they run first (for some weird reason), setting the PL version to `0.8`. Making the PennyLane build a wheel as well should make sure that it is indeed installed prior to any other plugin.

`Requirement already satisfied: pennylane from git+https://github.com/XanaduAI/pennylane.git@#egg=pennylane in ./venv/lib/python3.7/site-packages (from -r requirements.txt (line 13)) (0.8.0)`

Locally this change made the PL installation to be executed first
```suggestion
git+https://github.com/XanaduAI/pennylane.git
```

---

Good catch! Let's give this a go and see if it works.